app/section/helpers/generateOptions.js

```diff
@@ -1,11 +1,18 @@
 import { doesFormDataContainValue } from './formData';
 
 const populateQuestionOption = (
-  questionId, questionOption, formData,
+  questionId, questionOption, formData, questionType,
 ) => {
   const populatedOption = questionOption;
 
   if (doesFormDataContainValue(questionId, questionOption.value, formData)) {
+    if (questionType === 'combobox-options') {
+      return {
+        ...questionOption,
+        selected: true,
```

Comment:
so if the questionType is not 'combobox-options' then it has not been stored in db therefore is not selected?

Reply:
Not sure I know what you mean here. 

The main thing here is that for a combobox the component wants selected rather than checked. Which is why I've had to do this here.

Reply:
just wondering why all were being returned with selected: true

Reply:
I thought maybe we only stored the selected ones